src/components/Users/users.js

```diff
@@ -4,14 +4,12 @@ import React, { Component } from 'react';
 class Users extends Component {
   constructor(){
     super();
-    this.state = {
-      users:[]
-    }
+    this.state = {users:[]}
   }
 
   componentDidMount() {
-    fetch('/api/users')
-      .then(res => res.json())
+    fetch('http://localhost:5000/api/users')
+      .then(res => res.json(console.log(res)))
```

Comment:
Same comment as above related to `axios` :)

Reply:
ok